lusidtools/cocoon/async_tools.py

```diff
@@ -20,9 +25,16 @@ def stop_event_loop_new_thread(loop: asyncio.AbstractEventLoop) -> None:
     """
     Takes an event loop, stops it and once stopped closes it
 
-    :param asyncio.AbstractEventLoop loop: The loop to stop and close
-    :return: None
+    Parameters
+    ----------
+    loop : asyncio.AbstractEventLoop
+        The loop to stop and close
+
+    Returns
+    -------
+
```

Comment:
Is this the correct way to specify Returns when there is nothing returned from the function? I couldn't find any reference to what to do in this situation in the numpydoc docs.

Reply:
Interesting point, I did some digging and here it is suggested that all functions in python return *something*, either a specified value or **none** (which is technically something)

Ive posted on codecraft to see if anyone else has any thoughts 

This example uses google docstring format:
https://stackoverflow.com/questions/27784179/docstrings-when-nothing-is-returned

Reply:
After a poll on codecraft the consensus was to remove the section all together